### ISSUE TITLE
NuScenes dataset size

### ISSUE THREAD

**Issue by vobecant** [nutonomy/nuscenes-devkit#533]
Hi,

I have a question regarding the size of the dataset. On the webpage it says:

> The full dataset includes approximately 1.4M camera images, 390k LIDAR sweeps, 1.4M RADAR sweeps and 1.4M object bounding boxes in 40k keyframes.

However, when I use the dataloader provided, I can only see 34149 samples in 850 scenes of train+val. Am I looking at the wrong thing?

My problem is that I would like to have the access to those 1.4M images with RADAR sweeps and 390k LIDAR sweeps. Based on these numbers, I suppose that there is a RADAR sweep for every image but the same does not hold for LIDAR sweeps that are sparser in time. I would like to access each of these images and get RADAR sweep + LIDAR sweep (if available). I don't need to have any annotations and the calibration does not need to be perfect (as long as it is not considerably wrong).

Can you please help me?

Thank you very much for your help!

**Comment by holger-nutonomy** [comment]
Hi. We did not provide a "dataloader", but just the raw dataset at https://www.nuscenes.org/download. 
The statistics are either for train+val (850 scenes) or for train+val+test (1000 scenes). You can download the test set, just not the annotations.
On average there are ~40 samples per scene and the camera is recorded at 12Hz, radar at 13Hz and lidar at 20Hz. You need to take into account that there are 6 cameras, 5 radars and only 1 lidar. Take a look at the tutorial (https://www.nuscenes.org/nuscenes?tutorial=nuscenes). The parts that don't require annotations should work equally well for the test set.
